tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -172,6 +172,18 @@ def _similarity_search(cls, pinecone, pinecone_vector_store, embedding_model, qu
         LLMObs.disable()
         return mock_tracer.pop_traces()[0]
 
+    @classmethod
+    def _invoke_tool(cls, tool, tool_input, mock_tracer, cassette_name):
+        LLMObs.enable(ml_app=cls.ml_app, integrations_enabled=False, _tracer=mock_tracer)
+        if LANGCHAIN_VERSION > (0, 1):
+            if cassette_name is not None:
+                with get_request_vcr(subdirectory_name=cls.cassette_subdirectory_name).use_cassette(cassette_name):
+                    tool.invoke(tool_input)
```

Comment:
Correct me if I'm wrong, but the test tool we are invoking does not require an external web request/response since it is just a calculation tool. Do we need cassette testing here?

Reply:
True you are correct now that I think about it we don't need the cassettes in here.